README.md

```diff
@@ -1,223 +1,209 @@
+```
+python ~/repos/metashape/python/metashape_workflow.py ~/projects/forest_structure/metashape_configs/config001.yml
+python ~/repos/metashape/python/metashape_workflow.py ~/projects/forest_structure/metashape_configs/config002.yml
+python ~/repos/metashape/python/metashape_workflow.py ~/projects/forest_structure/metashape_configs/config003.yml
+```
 
-Here is a breakdown of the command:
+Then it's just a matter of running the shell script. 
 
-`docker run` is the command to run a docker image
+<br/>
 
-`-v </host/data/dir>:/data` is mounting a volume from your local computer into the container. We are mounting your directory that has the imagery and config file (</host/data/dir>) into the container at the path "/data".
+### Workflow outputs
 
-`-e AGISOFT_FLS=$AGISOFT_FLS` is declaring your floating license to use Metashape. We set the license info as an environmental variable earlier in these instructions (i.e., `export AGISOFT_FLS=<IP_address>:<port_number>`)
+The outputs of the workflow are the following:
+- **Photogrammetry outputs** (e.g., dense point cloud, orthomosaic, digital surface model, and Metashape processing report)
+- **A Metashape project file** (for additional future processing or for inspecting the data via the Metashape GUI)
+- **A processing log** (which records the processing time for each step and the full set of configuration parameters, for reproducibility)
 
-`--gpus all` If the container has access to your local GPUs, use this flag to enable it.
+The outputs for a given workflow run are named using the following convention: `{run_name}_{date_and_time}_abc.xyz`. For example: `set14-highQuality_20200118T1022_ortho.tif`. The run name and output directories are specified in the configuration file.
 
-`ghcr.io/open-forest-observatory/automate-metashape` This is the docker image that has the software to run the `automate-metashape` script. It is located in the Github container registry. When you execute the `docker run...` command, it will download the container image to your local machine and start the script to process imagery using Metashape. 
+<br/>
+<br/>
+<br/>
 
-#### Custom location of the Metashape configuration file
+---
 
-If your config.yaml is located anywhere other than `/data/config.yaml` (or the file is named differently), you can specify its location following one additional command line argument `--config_file` at the end of the `docker run` command. For example, if it is located in the container at `/data/configs/project_10/config.yml` (meanining, in the example above, it is located on your local computer at `~/drone_data/configs/project_10/config.yml`), just append `--config_file /data/configs/project_10/config.yml` to the `docker run` command above. So the command above would look like:
+<br/>
 
-`docker run -v </host/data/dir>:/data -e AGISOFT_FLS=$AGISOFT_FLS --gpus all ghcr.io/open-forest-observatory/automate-metashape --config_file /data/configs/project_10/config.yml`
+## Docker Workflow
```

Comment:
I would have initially expected the docker install to be in the same place as the conventional install. Were there too many differences between running in docker and bare metal to make this make sense?